test/icalendar/util/deserialize_test.exs

```diff
@@ -128,4 +128,12 @@ defmodule ICalendar.Util.DeserializeTest do
              %{"X-TITLE" => "Paris:geo:48.856788,2.351077", "X-ADDRESS" => nil}
            ]
   end
+
+  test "ignore empty lines" do
+    line = ""
+
+    kvs = Deserialize.retrieve_kvs(line)
+
+    assert kvs == nil
+  end
```

Comment:
Can we have a test that shows why there would be empty lines? i.e. a full ics string being parsed

Reply:
I included a sample from the VEVENT that was breaking my application.